SPECS/grub2/grub2.spec

```diff
@@ -103,6 +103,8 @@ Patch:          sbat-4-0003-fs-ntfs-Fix-an-OOB-read-when-parsing-directory-entri
 Patch:          sbat-4-0004-fs-ntfs-Fix-an-OOB-read-when-parsing-bitmaps-for-ind.patch
 Patch:          sbat-4-0005-fs-ntfs-Fix-an-OOB-read-when-parsing-a-volume-label.patch
 Patch:          sbat-4-0006-fs-ntfs-Make-code-more-readable.patch
+#CVE-2025-0624
```

Comment:
Nit-pick: I think the comment is not needed. It doesn't add more information and CVE patches are common in our codebase and well-understood.
```suggestion
```